scripts/add_existing_baseyear.py

```diff
@@ -617,6 +618,50 @@ def add_heating_capacities_installed_before_baseyear(
             )
 
 
+def add_ocgt_retro(n, baseyear, start):
+    """
+    Function to add OCGT H2 retrofitting of existing plants.
+    """
+    logger.info("Add OCGT H2 retrofitting.")
+    # repurposing cost of OCGT gas to H2 in % investment cost in EUR/MW
+    # source: Christidis et al (2023) - H2-Ready-Gaskraftwerke, Table 3
+    # https://reiner-lemoine-institut.de/wp-content/uploads/2023/11/RLI-Studie-H2-ready_DE.pdf
+    retro_factor_ocgt = 0.15
```

Comment:
it would be good to move these parameters to the config.yaml

Reply:
Good point. I added the configs `[H2_OCGT_retrofit][cost]` and `[H2_OCGT_retrofit][efficiency]`